projects/opendr_ws/src/perception/scripts/object_tracking_2d_deep_sort.py

```diff
@@ -174,36 +184,62 @@ def draw_predictions(frame, predictions: TrackingAnnotation, is_centered=False,
         )
 
 
-if __name__ == "__main__":
-    # Automatically run on GPU/CPU
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
-
-    # initialize ROS node
-    rospy.init_node("opendr_deep_sort", anonymous=True)
-    rospy.loginfo("Deep Sort node started")
-
-    model_name = rospy.get_param("~model_name", "deep_sort")
-    temp_dir = rospy.get_param("~temp_dir", "temp")
-    input_image_topic = rospy.get_param(
-        "~input_image_topic", "/opendr/dataset_image"
-    )
-    rospy.loginfo("Using model_name: {}".format(model_name))
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--model_name", help="Name of the trained model",
+                        type=str, default="deep_sort")
+    parser.add_argument("-t", "--temp_dir", help="Path to a temporary directory with models",
+                        type=str, default="temp")
+    parser.add_argument("-i", "--input_image_topic",
+                        help="Input Image topic provided by either an image_dataset_node, webcam or any other image node",
+                        type=str, default="/opendr/dataset_image")
+    parser.add_argument("-od", "--output_detection_topic",
+                        help="Output detections topic",
+                        type=str, default="/opendr/deep_sort_detection")
+    parser.add_argument("-ot", "--output_tracking_id_topic",
+                        help="Output detections topic",
+                        type=str, default="/opendr/deep_sort_tracking_id")
+    parser.add_argument("-oi", "--output_image_topic",
+                        help="Output detections topic",
```

Comment:
It would be helpful to add a more descriptive help text, right now it's the same as the detections topic, the docstring text from the class ctor would be ok.